src/components/dl-displayer-controller.test.tsx

```diff
@@ -82,25 +83,73 @@ describe('Distribution List Displayer Controller', () => {
 		expect(await screen.findByText(owners[9].name)).toBeVisible();
 	});
 
-	it('should render member list', async () => {
-		const members = times(10, () => faker.internet.email());
-		const dl = generateDistributionList();
-		registerGetDistributionListHandler(dl);
-		registerGetDistributionListMembersHandler(members);
-		setupTest(
-			<Route path={`${ROUTES.mainRoute}${ROUTES.distributionLists}`}>
-				<DLDisplayerController />
-			</Route>,
-			{
-				initialEntries: [
-					`/${ROUTES_INTERNAL_PARAMS.route.distributionLists}/${ROUTES_INTERNAL_PARAMS.filter.member}/${dl.id}`
-				]
+	describe('Member list', () => {
+		it.each(['FALSE', undefined])(
+			'should render member list if zimbraHideInGal is %s',
+			async (hideParam) => {
+				const members = times(10, () => faker.internet.email());
+				const dl = generateDistributionList();
+				registerGetDistributionListHandler({ ...dl, zimbraHideParam: hideParam as BooleanString });
```

Comment:
Remove cast in favor of generic of it.each

Reply:
@beawar is a <BooleanString | undefined> enough? Or is there a better way to define it?

Reply:
It is the one